webui/react/src/components/FilterForm/components/FilterFormStore.ts

```diff
@@ -34,12 +34,12 @@ const getInitGroup = (): FormGroup => ({
 });
 
 export const getInitField = (): FormField => ({
-  columnName: '',
+  columnName: 'id',
```

Comment:
Used id because thats the common column among the new experiment table, flat run table, search table.